### DISCUSSION TITLE
"Selectivity" in metrics export

### DISCUSSION THREAD

**Discussion by neatdot** [DaxStudio/DaxStudio#598] [Q&A]
The Export Metrics option includes an attribute called "Selectivity" which is expressed as a percentage. Can anyone explain what this attribute represents and how it is calculated?

Thanks
Steve

**Comment by marcosqlbi** [comment]
It is the ratio between the column and table cardinality. For example, a column that has 10 unique values in a table of 1,000 rows has a selectivity of 0.01 - I wouldn't say that it's a percentage. Its maximum value is 1 in case the column is a unique identifier for the table.

**Reply by neatdot** [reply]
Thanks, Marco.
